onnx/test/helper_test.py

```diff
@@ -410,12 +410,14 @@ def test(opset_versions: list[tuple[str, int]], ir_version: int) -> None:
         test([("", 19)], 9)
         test([("", 20)], 9)
         test([("", 21)], 10)
+        test([("", 22)], 10)
         # standard opset can be referred to using empty-string or "ai.onnx"
         test([("ai.onnx", 9)], 4)
         test([("ai.onnx.ml", 2)], 6)
         test([("ai.onnx.ml", 3)], 8)
         test([("ai.onnx.ml", 4)], 9)
         test([("ai.onnx.ml", 5)], 10)
+        test([("ai.onnx.ml", 6)], 10)
```

Comment:
Is there a new ai.onnx.ml opset? I haven't been following the 1.17 changes so I'm not sure. 

The info I can gleen from the code I see seems inconsistent:
* https://onnx.ai/onnx/operators/index.html (which I believe is based on `main`) on the `ai.onnx.ml` tab, I only see up to `5`.
* https://github.com/onnx/onnx/blob/main/onnx/helper.py#L80 does show a `6`

You might need to reach out to the Operators team to get clarity and either get the documentation or helper.py updated.

Reply:
No, I don't think there have been any updates to ai.onnx.ml opset ... eg., [this file](https://github.com/onnx/onnx/blob/99e1fd352c05c3176770080824fd7a8c474c97c0/onnx/defs/operator_sets_ml.h#L105)  is the source for onnxml opsets

Reply:
I reviewed all the commits from previous release upto now, there is no reference to update to ai.onnx.ml. I had updated the above helper_test.py as there was an update to [helper.py](https://github.com/onnx/onnx/blob/main/onnx/helper.py#L80) in this PR : [6099](https://github.com/onnx/onnx/pull/6099)

I will follow up [thiagocrepaldi](https://github.com/thiagocrepaldi) to check if this was expected.

Reply:
Confirmed with [thiagocrepaldi](https://github.com/thiagocrepaldi) that the opset number in this file can be rolled back.

Reply:
Resolving comment as changes are reverted